sql/addons_view_v1.sql

```diff
@@ -0,0 +1,15 @@
+SELECT
+  @submission_date as submission_date,
+  document_id,
+  client_id,
+  sample_id,
+  subsession_start_date,
+  normalized_channel,
+  addons_array.element.*
+FROM
+  telemetry.main_summary_v4
+CROSS JOIN UNNEST(active_addons.list) as addons_array
+WHERE
+  submission_date = @submission_date AND
+  client_id IS NOT NULL
```

Comment:
this should be formatted using the [bigquery console](https://console.cloud.google.com/bigquery), which puts `AND` at the beginning of lines, rather than the end.

additional context: we would be following the [SQL Style Guide](https://docs.telemetry.mozilla.org/concepts/sql_style.html?highlight=sql#code-blocks), but auto-formatting is more effective than manually enforced style, so until we figure out how to assert style in CI we are defaulting to the bigquery console's mechanical formatting.